solr/core/src/java/org/apache/solr/util/SolrPluginUtils.java

```diff
@@ -678,7 +678,17 @@ static int calculateMinShouldMatch(int optionalClauseCount, String spec) {
       spec = spaceAroundLessThanPattern.matcher(spec).replaceAll("<");
       for (String s : spacePattern.split(spec)) {
         String[] parts = lessThanPattern.split(s,0);
-        int upperBound = Integer.parseInt(parts[0]);
+        if (parts.length == 0) {
+          throw new SolrException(SolrException.ErrorCode.BAD_REQUEST,
+              "Operator < must be followed by a number");
+        }
+        int upperBound;
+        try {
+          upperBound = Integer.parseInt(parts[0]);
```

Comment:
could you please handle [this](https://github.com/apache/lucene-solr/pull/978/files#diff-72f9d854f80b7cfab1f2da1c7bdcb607R707) too?

Reply:
That comment seems to just link to itself.

Reply:
Done in 5442fde, and the case with neither '<' nor '%' is handled by 6cac06f.